src/looprestoration.rs

```diff
@@ -294,3 +298,106 @@ pub(crate) unsafe fn padding<BD: BitDepth>(
         }
     };
 }
+
+// TODO(randompoison): Temporarily public until init logic is deduplicated.
+pub(crate) unsafe extern "C" fn wiener_erased<BD: BitDepth>(
+    mut p: *mut libc::c_void,
+    stride: ptrdiff_t,
+    left: *const libc::c_void,
+    mut lpf: *const libc::c_void,
+    w: libc::c_int,
+    h: libc::c_int,
+    params: *const LooprestorationParams,
+    edges: LrEdgeFlags,
+    bitdepth_max: libc::c_int,
+) {
+    let bd = BD::new(bitdepth_max.as_());
+    wiener_rust::<BD>(
+        p.cast(),
+        stride,
+        left.cast(),
+        lpf.cast(),
+        w,
+        h,
+        params,
+        edges,
+        bd,
+    )
+}
+
+unsafe extern "C" fn wiener_rust<BD: BitDepth>(
```

Comment:
We can drop the `extern "C"` now.  The `*_rust` `fn`s shouldn't need to be `extern "C"` at all.  You could save this for a PR cleaning up `wiener_rust`, though, or just do it now.

Reply:
Done!